packages/web-shared/ui-kit/ContentCard/ContentCard.js

```diff
@@ -37,6 +37,7 @@ function ContentCard({
       backgroundColor="neutral.gray6"
       height="100%"
       display={horizontal ? 'flex' : ''}
+      onClick={onClick}
```

Comment:
@nlewis84 this change is intentional - I'm fixing a regression where clicking on cards wouldn't navigate